presto-hive/src/main/java/com/facebook/presto/hive/InternalHiveSplit.java

```diff
@@ -241,7 +243,7 @@ public int getEstimatedSizeInBytes()
     public String toString()
     {
         return toStringHelper(this)
-                .add("path", path)
+                .add("path", relativeUri)
```

Comment:
Also let's convert it to string here.

Reply:
relativeUri is not a byte array. byte array's toString doesn't print anything reasonable (just an identity hashcode of that bytearray. I would suggest converting this bytearray to string when adding it to the `toStringHelper`